src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderProgramProfile.java

```diff
@@ -42,8 +46,11 @@ public boolean isCurrentWorkActivity() {
             && (endDate == null || endDate.isAfter(currentDate));
         final var isValidCurrentActivity = courseActivity != null &&
             courseActivity.getScheduleStartDate() != null && courseActivity.getScheduleStartDate().isBefore(currentDate.plusDays(1))
-            && (courseActivity.getScheduleEndDate() == null || courseActivity.getScheduleEndDate().isAfter(currentDate))
-            && courseActivity.getCode() != null && !courseActivity.getCode().startsWith("EDU");
-        return isCurrentProgramProfile && isValidCurrentActivity;
+            && (courseActivity.getScheduleEndDate() == null || courseActivity.getScheduleEndDate().isAfter(currentDate));
```

Comment:
Definitely a matter of opinion but would using optional make this look a bit neater?